include/freerdp/input.h

```diff
@@ -105,6 +107,8 @@ extern "C"
 	                                                           UINT16 code);
 	FREERDP_API BOOL freerdp_input_send_mouse_event(rdpInput* input, UINT16 flags, UINT16 x,
 	                                                UINT16 y);
+	FREERDP_API BOOL freerdp_input_send_rel_mouse_event(rdpInput* input, UINT16 flags, INT16 x,
+	                                                    INT16 y);
```

Comment:
`delta_x`, `delta_y` would fit better here (or camel case equivalent) to things more understandable upon reading